src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -217,6 +263,216 @@ await contextualFilter.EvaluateAsync(context, appContext).ConfigureAwait(false)
             return enabled;
         }
 
+        public ValueTask<Variant> GetVariantAsync(string feature, CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrEmpty(feature))
+            {
+                throw new ArgumentNullException(nameof(feature));
+            }
+
+            return GetVariantAsync(feature, null, false, cancellationToken);
```

Comment:
Can you use named parameters for the places where literals are passed? It helps readability.